src/placeos-driver/interface/sensor.cr

```diff
@@ -166,7 +168,7 @@ module PlaceOS::Driver::Interface::Sensor
     property y : Float64
   end
 
-  abstract class Detail
+  class Detail
```

Comment:
Is there any reason for this object to be muteable and/or a class?

Reply:
yeah, I expect it to be inherited so additional fields can be added
but area management only cares about the base class for summaries however the additional fields will be preserved and included in influxdb